src/js/frontend/src/components/ChartTitle.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import Radium from "radium"
+import { Link } from 'react-router'
+
+let ChartTitle = React.createClass({
```

Comment:
I mean more that if there's no need for it the component shouldn't be a class at all, just a function so this would actually be: 
```
function ChartTitle (props) {
 return (<div>Content here</div>)
}
```
Since it's a "dumb" component and requires no state handling or lifecycle actions